lib/rules/one-var.js

```diff
@@ -12,25 +13,58 @@
 
 module.exports = function(context) {
 
+    var MODE = context.options[0] || "always";
+
     //--------------------------------------------------------------------------
     // Helpers
     //--------------------------------------------------------------------------
 
     var functionStack = [];
 
+    /**
+     * Increments the functionStack counter.
+     * @returns {undefined} No return value.
+     * @private
+     */
     function startFunction() {
         functionStack.push(false);
     }
 
+    /**
+     * Decrements the functionStack counter.
+     * @returns {undefined} No return value.
+     * @private
+     */
     function endFunction() {
         functionStack.pop();
     }
 
-    function checkDeclarations(node) {
+    /**
+     * Determines if there is more than one var statement in the current scope.
+     * @param {ASTNode} node The node to check.
+     * @returns {Boolean} Returns true if it is the first var declaration, false if not.
+     * @private
+     */
+    function checkOnlyOneVar() {
```

Comment:
I think a better name is `hasOnlyOneVar()` if you reverse the return values. That way, you don't have to negate the result.